esp/xslt/esdl2monitor.xslt

```diff
@@ -203,32 +204,47 @@ END;
     string100 Message {xpath('Message')};
   END;
 
-  soapoutRec := record
+  soapoutRec := record <xsl:choose><xsl:when test="$platform='ESP'">(the_responseLayout)
```

Comment:
it's a minor stylistic concern, no reason to play tricks to fix it since it affects the output.